home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -498,7 +498,132 @@ paths:
+      responses:
+        201:
+          $ref: '#/components/responses/TaskResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+    get:
+      tags:
+        - cooperation
+      summary: TODO
+      description: TODO
+      operationId: getTasks
+      parameters:
+        - $ref: '#/components/parameters/p_id'
+      responses:
+        201:
+          $ref: '#/components/responses/TasksResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+  /cooperation/{id}/tasks/{task_id}:
+    get:
+      tags:
+        - cooperation
+      summary: TODO
+      description: TODO
+      operationId: getTask
+      parameters:
+        - $ref: '#/components/parameters/p_id'
+        - $ref: '#/components/parameters/p_task_id'
+      responses:
+        201:
+          $ref: '#/components/responses/TaskResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+  /cooperation/{id}/tasks/{task_id}/file:
+    get:
+      tags:
+        - cooperation
+      summary: TODO
+      description: TODO
+      operationId: getFile
+      parameters:
+        - $ref: '#/components/parameters/p_id'
+        - $ref: '#/components/parameters/p_task_id'
+      responses:
+        201:
+          $ref: '#/components/responses/XlsxFileResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+    post:
+      tags:
+        - cooperation
+      summary: TODO
+      description: TODO
+      operationId: createFile
+      parameters:
+        - $ref: '#/components/parameters/p_id'
+        - $ref: '#/components/parameters/p_task_id'
+      requestBody:
+        $ref: '#/components/requestBodies/ExcelBody'
+      responses:
+        201:
+          $ref: '#/components/responses/NoContentResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+  /cooperation/{id}/tasks/{task_id}/report:
```

Comment:
Do we want to name it a report? Can we find a different name for this data? What about `summary`?